examples/end-to-end-examples/stable_diffusion/requirements.txt

```diff
@@ -10,6 +10,7 @@ omegaconf==2.3.0
 # -------- diffusion -------- #
 diffusers[torch]==0.11.1
 transformers[torch]==4.25.1
+datasets>=2.14.6
```

Comment:
might as well pin this to an exact datasets version? (same as other places)

Reply:
done